jerry-core/api/jerry.c

```diff
@@ -4362,15 +4362,24 @@ jerry_get_promise_state (const jerry_value_t promise) /**< promise object to get
  * Note:
  *     the previous callback is overwritten
  */
-void jerry_promise_set_callback (jerry_promise_callback_t callback,  /**< notification callback */
+void jerry_promise_set_callback (jerry_promise_event_filter_t filters, /**< combination of event filters */
+                                 jerry_promise_callback_t callback, /**< notification callback */
                                  void *user_p) /**< user pointer passed to the callback */
 {
   jerry_assert_api_available ();
 
 #if JERRY_BUILTIN_PROMISE && JERRY_PROMISE_CALLBACK
+  if (filters == JERRY_PROMISE_EVENT_FILTER_DISABLE || callback == 0)
```

Comment:
minor thing: Should we use `callback == NULL` instead of a simple zero, to indicate that the callback is a pointer?